.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - '6'
+  - '8'
```

Comment:
It just occurred to me that we can solve this using yarn resolutions and stick with node 6 for now. If we add

```
"**/sort-package-json": "1.23.0"
```

the the resolutions section of package.json, the dependencies will all install on node 6.

Reply:
Eh, never mind. This would technically work, but I realized that in my PR I'm going to need to build on node 8 anyway because to build against Ember 3.12+ I need to use the 1.0.0 of `ember-concurrency-decorators` and it doesn't support Node 6. So may as well jump to 8 here.

Reply:
Ya I think we can leave this for now. Resolutions is a good idea until you forget to remove them, which renders dependabot effectively useless for that dependency.